h/static/styles/sidebar-tutorial.scss

```diff
@@ -48,3 +50,8 @@
                       first list item (i.e. between the list and whatever's
                       above it). */
 }
+
+
+@include icons {
+  font-family: 'h';
+}
```

Comment:
We don't need this. Any class which begins with h-icon-, gets a font-family of h.
i.e [class^="h-icon-"], [class*=" h-icon-"]